### PR TITLE
finally include the power consumption

### PR DESCRIPTION
**Pull request by kreier**
It's been a while

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -101,12 +101,17 @@ As [measured in April 2020](https://github.com/kreier/solarmeter/blob/master/REA
 
 Further measurement has been conducted in November 2020. The results.
 
+- Running 66 mA
+- WiFi 80-140 mA
+- CPU 80 MHz 48 mA
+- Light Sleep 9 mA
+
```
Comment (kreier) [inline docs/index.md:108]: How much deep sleep mA?

File: docs/index.md [hunk 2]
```diff
@@ -101,12 +101,17 @@ As [measured in April 2020](https://github.com/kreier/solarmeter/blob/master/REA
...
 You can directly measure the voltage of the LiPo battery on Pin 34. This can be seen in the [provided schematics from TTGO for the T-Display](TTGO_T-Display_schematics.pdf):
 
 <img src="bat_esp32.png" width="65%" align="right">
 <img src="bat_divider.png" width="25%">
 
-The pins on the T-Display are
+The pin on the T-Display is 34 (channel ADC1_CH6) with a 1:2 battery voltage divider.
```
Comment (kreier) [inline docs/index.md:114]: voltage to convert from digital value

File: docs/index.md (hunk 1 above)
Comment (ssis-robotics) [reply docs/index.md:108]: The tested ones still had something in the mA range. Need to check the different boards.

File: docs/index.md (hunk 2 above)
Comment (kreier) [reply docs/index.md:114]: Please compare later to the T8 ESP32-S2 ST7789

File: docs/index.md [hunk 3]
```diff
@@ -101,12 +101,17 @@ As [measured in April 2020](https://github.com/kreier/solarmeter/blob/master/REA
 
 Further measurement has been conducted in November 2020. The results.
 
+- Running 66 mA
+- WiFi 80-140 mA
+- CPU 80 MHz 48 mA
+- Light Sleep 9 mA
```
Comment (ssis-robotics) [inline docs/index.md:107]: There might be some issues with deep sleep. The power consumption seems not to drop.

File: docs/index.md (hunk 2 above)
Comment (ssis-robotics) [reply docs/index.md:114]: Will be finished in January 2022

**Pull request merged**